docs/source/roadmap.md

```diff
@@ -0,0 +1,287 @@
+
+Nevertheless, to demonstrate _starfish_'s scale, we will implement workflows on one or more existing pipeline runner and keep this work distinct from the underlying library. This pipeline runner will be capable of processing a complete experiment consisting of arbitrary numbers of fields of view for any SpaceTx lab and will leverage cloud-based parallelism to accomplish the processing of any SpaceTx experiment in under 2 hours. We will consider the HCA use case when designing our version 0.1.0 workflows in order to maximize initial compatibility.
+
+### A Developer Focus Point
+Two key aims of _starfish_ and SpaceTx are to (1) encourage convergence of the community onto a single object model for processing of image-based transcriptomics data and (2) make it easier to develop pipelines by providing a high-performance platform. To facilitate this, we must encourage developers to work on _starfish_ through active outreach and by being encouraging and helpful.
+
+## Outstanding work
+
+### A General Format and Object Model for Image-based Transcriptomics
+
+#### Format specification for _starfish_ outputs, IntensityTable and ExpressionMatrix:
+These two objects currently lack format specifications, which has confused some users. A specification for these formats will help computational users to understand how to consume our data formats, and will help facilitate their conversion for use in other languages. For users who are comfortable with python but not necessarily with xarray, we will build in a series of `to_array` and `to_pandas` options.
+
+#### _starfish_ must enable processing of fields of view by round
+Sequential smFISH assays that capture volumetric images produce fields of view that are too large for personal computers<sup>[7](#fn7)</sup>. However, if we are able to break up the processing of this data by round, we could hit our scale targets. Second, multiplexed assays (e.g. MERFISH, SeqFISH, ISS) require all rounds and channels to be loaded into memory to decode. In both cases, we should derisk decoding of these assays by understanding the maximum ImageStack size we can support before we need to implement solutions to manage memory consumption.
+
+### A Modular Library of Pipeline Components
+
+#### Documentation
+We need to add basic documentation that describes how to use each component, and how to fit parameters. This should be adequate for a skilled computational user to pick up _starfish_ and use it to make a pipeline. _starfish_ should also clearly document how a developer can contribute code they need for their analyses to the project. We will need to source feedback from users on what parts of the documentation provoke confusion.
+
+#### Image re-scaling & normalization
```

Comment:
Is reslicing existing data sets and persisting them onto disk something we need to do?  We now have a 3000px cap on the tiles, which may or may not be an issue.

Reply:
I think this is best answered by looking at the sizes of the fields of view we are being given by each contributor and the requirement that they need to run on a laptop. So long as the laptop requirement is fulfilled and the library is fast, the implementation/execution is up to you.